src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -325,11 +324,6 @@ func (updated *Config) applyFlags(flags map[string]interface{}) error {
 			updated.CloudConfigCA = value.(string)
 		case "instanceid":
 			updated.Client.DeviceID = value.(string)
-
-		// HTTP-server
-		case "uiaddr":
```

Comment:
There's no need for uiaddr to be in the config, and even stranger I noticed that we were pushing updates to it via the cloud.yaml.  uiaddr now just comes from flags at runtime.